platform/jvm/gradle/libs.versions.toml

```diff
@@ -19,7 +19,7 @@ mockitoCore = "4.9.0"
 mockitoKotlin = "2.2.0"
 mockitoKotlinVersion = "4.1.0"
 okhttp = "4.12.0"
-robolectric = "4.9"
```

Comment:
Until we finish the migration from bazel to gradle the integration (robolectric) tests are mostly run via CI using bazel, which means you might need to also bump https://github.com/bitdriftlabs/capture-sdk/blob/a5e8d0fbd98d219be8998148ff31e5b5cc0bc3d7/bazel/capture_dependencies.bzl#L21

As you can tell the versions were already out of sync :(

Reply:
Great catch! I tested both Bazel (`./bazelw test //platform/jvm/...`) and Gradle locally. I was lucky that Robolectric's Bazel version was set to `4.11` and everything was passing 😅 (refs. https://github.com/bitdriftlabs/capture-sdk/pull/24#discussion_r1744750393). Will bump Bazel dependency version to `4.13` to have them in sync 👍

Reply:
> Will bump Bazel dependency version to 4.13 to have them in sync 👍

Done 👉 https://github.com/bitdriftlabs/capture-sdk/pull/24/commits/b3ac74c4cd4cd66ef0fc9d1469a5740447ee411c